### PR TITLE
feat: Add native select control

### PR DESCRIPTION
**Pull request by istarkov**
## New

- Native Select

## Changes

- Improved detection algorithm to include invisible elements (like `option` or `display:none`).
- Added interceptor to handle pointer down events, preventing select from opening.
- Enhanced element selection on canvas to support `display:none` and option selection.
- Fixed prop types generation (previously, the value property was broken).
- Memory props has effect on canvas only

## Steps for reproduction

Add `Select` see it works

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env` file

### CODE REVIEW

File: packages/sdk-components-react/src/components.ts [hunk 1]
```diff
@@ -32,3 +32,5 @@ export { VimeoPlayButton } from "./vimeo-play-button";
 export { VimeoSpinner } from "./vimeo-spinner";
 export { XmlNode } from "./xml-node";
 export { Time } from "./time";
+export { Select } from "./select";
+export { Option } from "./option";
```
Comment (TrySound) [inline packages/sdk-components-react/src/components.ts:36]: Do we need them separate? Or could be grouped like radix components?

Comment (istarkov) [reply packages/sdk-components-react/src/components.ts:36]: Props are not generated in a single file

**Comment by kof** [comment]
- [ ] Prop value needs to be in initial props after name
- [ ] Prop multiple needs to be in initialProps
- [ ] Style preset on option don't match rendering when prop multiple is used, e.g. background color is transparent in style panel but is dark gray on canvas
- [ ] when using json value array with prop multiple, validation fails, because it thinks value can only be a string, but it can take an array too

File: packages/sdk-components-react/src/components.ts (hunk 1 above)
Comment (TrySound) [reply packages/sdk-components-react/src/components.ts:36]: How so?
image

**Pull request closed without merging**